pkg/trace/schema/tables.go

```diff
@@ -0,0 +1,41 @@
+package schema
+
+import "github.com/tendermint/tendermint/config"
+
+func init() {
+	config.DefaultInfluxTables = AllTables()
+}
+
+func AllTables() []string {
```

Comment:
A func description would be helpful here; specifically, what does the string slice contain?